packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -62,17 +113,37 @@ export class ClusterParameterGroup extends ClusterParameterGroupBase {
    */
   public readonly clusterParameterGroupName: string;
 
+  /**
+   * The parameters in the parameter group
+  */
+  readonly parameters: { [name: string]: string };
+
   constructor(scope: Construct, id: string, props: ClusterParameterGroupProps) {
     super(scope, id);
-
+    this.parameters = props.parameters;
     const resource = new CfnClusterParameterGroup(this, 'Resource', {
       description: props.description || 'Cluster parameter group for family redshift-1.0',
       parameterGroupFamily: 'redshift-1.0',
-      parameters: Object.entries(props.parameters).map(([name, value]) => {
-        return { parameterName: name, parameterValue: value };
-      }),
+      parameters: Lazy.any({ produce: () => this.parseParameters() }),
```

Comment:
Why `Lazy` here as well?

Reply:
I thought `Lazy` had to be used if the parameters could be added after construct instantiation